modules/ROOT/pages/message-flows.adoc

```diff
@@ -19,55 +19,71 @@ Send messages to trading partners. An outbound message flow receives application
 
 Create an inbound message flow to receive EDI transactions from your trading partner, transform, and then send the translated application message to your backend systems.
 
-The following diagram shows the inbound message configuration building blocks of an inbound message flow and how the Partner Manager’s message processing framework orchestrates the end-to-end routing and transformation of your inbound EDI transactions from your trading partners to your backend systems at runtime.
+The following diagram shows the inbound message configuration of an inbound message flow. The diagram also shows how Partner Manager’s message processing framework orchestrates the end-to-end routing and transformation of your inbound EDI transactions from your trading partners to your backend systems at runtime.
 
 image::inbound-message-flow.png[Inbound message flow configuration building blocks diagram]
 
-. Receiving endpoint +
-Endpoint where you (the host) receive transmissions from the trading partner. Supported protocols include:
-* AS2 
+|===
+|Setting |Description |Supported Formats/Protocols
```

Comment:
GLOBALLY replace / with the best of either "and" or "or"
```suggestion
|Setting |Description |Supported Formats or Protocols
```

Reply:
Done